CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+
+```jsonc
+{
+  "version": 1,
+  "type": "ballotNote",
+  "dRepId": "b2f0...20fb",
+  "proposalId": "abcd1234+0",
+  "voteChoice": "Yes",
+  "rationale": "Supports Cardano's long-term growth",
+  "timestamp": "2025-03-10T12:00:00Z",
+  "anchorUrl": "ipfs://proposalDoc",
+  "anchorHash": "blake2b256:def456..."
+}
+```
+
+#### 4.3 Endorsement
+
+**Minted by:** Any stakeholder (SPO, DAO, community member) wishing to show support for a DRep.  
+**Purpose:**
+
+- On-chain “vote of confidence,” optionally with a comment or identity proof.
+- May include a **minimum ADA** amount to discourage spam.
+- Optionally allows endorsers to attach **additional** ADA to assist new DReps (e.g., raising the 500 ADA deposit) or provide financial backing.
+
+**Core Fields:**
+
+- `type = "endorsement"`
+- `endorses` = CIP-1694 DRep ID
+- `endorserStakeKey` (the minter’s stake key)
+
+**Optional Fields:**
+
+- `identityProof` (e.g., a social media link)
+- `comment` (why the endorser supports this DRep)
+- `timestamp` (ISO-8601)
+- **Financial Contribution**: Some platforms **may** store or display the ADA attached to this NFT (e.g., `"attachedAda": "<lovelaceAmount>"`) if relevant.
+- CIP-119 / CIP-108 anchor for more detailed statements.
+
+**Minimum or Additional ADA Considerations**
+
+- **Minimum ADA**: A platform can require a base amount (e.g., 2 ADA) to mint the endorsement NFT, helping deter zero-cost spam.
+- **Extra ADA**: Endorsers can optionally contribute a higher amount (e.g., 10 ADA, 500 ADA) to support the DRep’s deposit or other governance activities.
+- **Implementation**: Each wallet or dApp **decides** how to handle this attached ADA. For instance, they might introduce tiers of endorsements, each with a distinct cost/benefit.
+- **On-Chain Recording**: While the attached ADA is part of the transaction, the NFT’s datum could include a field (`"attachedAda"`) indicating the actual lovelace contributed.
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "type": "endorsement",
+  "endorses": "b2f0...20fb",
+  "endorserStakeKey": "stake1u9xyz...",
+  "identityProof": "https://twitter.com/endorserProfile",
+  "comment": "I trust this DRep’s track record",
+  "timestamp": "2025-03-15T09:00:00Z"
+
+  // If relevant, a field could note how much ADA was attached:
+  // "attachedAda": "10000000" // (10 ADA in lovelace)
+}
+```
+
+### 5. Implementation Guidelines
+
+1. **Datum Format (CIP-68)**
+
+   - Each NFT **must** include a Plutus datum referencing the CIP-1694 IDs and any optional fields above.
+   - CIP-68’s reference token mechanism can allow updates if desired (not mandated by this CIP).
+
+2. **Asset Naming & Labels (CIP-67)**
+
+   - Use the same CIP-67 label for these governance NFTs.
+   - Actual label allocation requires a **CIP-67 registry entry**.
+   - Tools can rely on this label for discovery and classification.
+
+3. **Reference Integrity**
+
+   - Check that `dRepId` and `proposalId` are valid CIP-1694 hashes/IDs.
```

Comment:
what should tools do IF some of this is not valid?

Reply:
Tools can keep operating as it has in the past. This is internal to the collective.

Reply:
> This is internal to the collective.

okay cool, I would suggest maybe it does not need to be a CIP if this is for internal tooling